sass/_Quickview.scss

```diff
@@ -20,23 +20,42 @@
     z-index: 1;
     > div {
       border: solid;
-      border-color: $color-greyish-dark-blue transparent;
       content: '';
       position: absolute;
       z-index: 99;
-      left: calc(50% - 6px);
     }
 
     // Tooltip under element, arrow on top
     &[x-placement^='bottom'] > div {
       top: -6px;
+      left: calc(50% - 6px);
       border-width: 0 6px 7px;
+      border-color: $color-greyish-dark-blue transparent;
     }
 
     // Tooltip over element, arrow on bottom
     &[x-placement^='top'] > div {
       bottom: -6px;
+      left: calc(50% - 6px);
       border-width: 7px 6px 0;
+      border-color: $color-greyish-dark-blue transparent;
+    }
+
+    // Tooltip on the right of element, arrow on the left
+    &[x-placement^='right'] > div {
+      top: calc(50% - 5px);
+      left: -6px;
+      border-width: 6px 7px 6px 0;
+      border-color: transparent $color-greyish-dark-blue;
+    }
+
+    // Tooltip on the left of element, arrow on the right
+    &[x-placement^='left'] > div {
+      top: calc(50% - 5px);
+      right: -6px;
+      left: auto;
+      border-width: 6px 0 6px 7px;
```

Comment:
Why do we fix the border ?

Reply:
Arrow size, it's made with borders, I should put it inside a variable tough